docs/reference/api/docker_remote_api_v1.15.md

```diff
@@ -562,6 +564,7 @@ Json Parameters:
       a boolean value.
 -   **Dns** - A list of dns servers for the container to use.
 -   **DnsSearch** - A list of DNS search domains
+-   **DnsOptions** - A list of DNS options
```

Comment:
I don't think this PR should affect API versions < v1.21, if it does, should the code be updated to perform a version-check before accepting/outputting this?